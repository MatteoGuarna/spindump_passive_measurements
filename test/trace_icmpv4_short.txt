Two short ICMP echo-echo response pairs using IPv4.
