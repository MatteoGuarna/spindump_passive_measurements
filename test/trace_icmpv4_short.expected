ICMP 31.133.149.35 <-> 212.16.98.51 48895 at 1553417873690045 new starting packets 0 0 bytes 0 0
ICMP 31.133.149.35 <-> 212.16.98.51 48895 at 1553417873728020 measurement up right 37975 packets 1 0 bytes 84 0 bandwidth 84 0
ICMP 31.133.149.35 <-> 212.16.98.51 48895 at 1553417874732523 measurement up right 37859 packets 2 1 bytes 168 84 bandwidth 84 84
